modules/data-entry/src/main/java/io/uhndata/cards/DeleteServlet.java

```diff
@@ -374,6 +380,7 @@ private String listReferrersFromTraversal()
             addNodesToResult(results, "subject", subjects);
             addNodesToResult(results, "subject type", subjectTypes);
             addNodesToResult(results, "questionnaire", questionnaires);
+            addNodesToResult(results, "answer", answerCount);
```

Comment:
And let's add a label for it
```suggestion
            addNodesToResult(results, "answer", answerCount);
            addNodesToResult(results, "answe sectionr", answerSectionCount);
```

Reply:
Right, I see, thanks